Bottom: d0ee2a91bf52996cd27ecee64e40fc40f4f8f55b
Top:    0b8167aaf50246bbdb74d12b8f4ca4d34ba955e7
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-23 02:37:12 +0000

Add FoundColor to resource settings, distinct from the Connected color


---

diff --git a/src/create.c b/src/create.c
index 992c49e..8b5c2b9 100644
--- a/src/create.c
+++ b/src/create.c
@@ -105,6 +105,7 @@ pcb_colors_from_settings (PCBType *ptr)
 
   /* copy default settings */
   ptr->ConnectedColor = Settings.ConnectedColor;
+  ptr->FoundColor = Settings.FoundColor;
   ptr->ElementColor = Settings.ElementColor;
   ptr->RatColor = Settings.RatColor;
   ptr->InvisibleObjectsColor = Settings.InvisibleObjectsColor;
diff --git a/src/global.h b/src/global.h
index 7b966dc..91cb74a 100644
--- a/src/global.h
+++ b/src/global.h
@@ -505,7 +505,7 @@ typedef struct PCBType
     *InvisibleObjectsColor,
     *InvisibleMarkColor,
     *ElementSelectedColor,
-    *RatSelectedColor, *ConnectedColor, *WarnColor, *MaskColor;
+    *RatSelectedColor, *ConnectedColor, *FoundColor, *WarnColor, *MaskColor;
   long CursorX,			/* cursor position as saved with layout */
     CursorY, Clipping;
   Coord Bloat,			/* drc sizes saved with layout */
@@ -636,6 +636,7 @@ typedef struct			/* some resources... */
     *ElementSelectedColor,
     *RatSelectedColor,
     *ConnectedColor,
+    *FoundColor,
     *OffLimitColor,
     *GridColor,
     *LayerColor[MAX_LAYER],
diff --git a/src/main.c b/src/main.c
index 3ab4d56..d80612d 100644
--- a/src/main.c
+++ b/src/main.c
@@ -760,12 +760,22 @@ Color of selected rats. Default: @samp{#00ffff}
 /* %start-doc options "3 Colors"
 @ftable @code
 @item --connected-color <string>
-Color to indicate connections. Default: @samp{#00ff00}
+Color to indicate physical connections. Default: @samp{#00ff00}
 @end ftable
 %end-doc
 */
   COLOR (ConnectedColor, "#00ff00", "connected-color",
-	 "color to indicate connections"),
+	 "color to indicate physically connected objects"),
+
+/* %start-doc options "3 Colors"
+@ftable @code
+@item --found-color <string>
+Color to indicate logical connections. Default: @samp{#ff00ff}
+@end ftable
+%end-doc
+*/
+  COLOR (FoundColor, "#ff00ff", "found-color",
+	 "color to indicate logically connected objects"),
 
 /* %start-doc options "3 Colors"
 @ftable @code
